gradle-plugins/src/main/java/com/linkedin/pegasus/gradle/PegasusPlugin.java

```diff
@@ -527,6 +527,7 @@
   // by default it is disabled, because it triggers hot-reload (even if it results in a no-op)
   private static final String DESTROY_STALE_FILES_ENABLE = "enableDestroyStaleFiles";
   public static final Collection<String> DATA_TEMPLATE_FILE_SUFFIXES = new ArrayList<>();
+  private static final String DATA_TEMPLATE_ENABLE_ARGFILE = "generateDataTemplate.enableArgFile";
```

Comment:
Is this necessary? Task inputs can be configured directly as such:
```
generateMainDataTemplate {
  enableArgFile = true
}
```
This is how the fix for `generateXXXRestClient` is enabled.

Reply:
Remove the property based logic. We can enable this using task args directly.